### PR TITLE
[geolocator] Implement getLastKnownPosition

### PR DESCRIPTION
**Pull request by bbrto21**
* change project type to staticLib

Signed-off-by: Boram Bae <boram21.bae@samsung.com>

### CODE REVIEW

**Review by HakkyuKim — APPROVED** [review]
LGTM on implementations.

Could you sync with the changes done in https://github.com/flutter-tizen/plugins/pull/175/files?
- Removing `TIZEN_DEPRECATION DEPRECATION_WARNING` from `USER_CPP_DEFS`.
- Removing "/linker" from "# Compiler/linker flags".
- Removing lines from `USER_CPPFLAGS_MISC` to `USER_OBJS` and replacing with just `USER_CPPFLAGS_MISC =`.

File: packages/geolocator/tizen/src/locaton_manager.cc [hunk 1]
```diff
@@ -62,6 +62,36 @@ TizenResult LocationManager::RequestCurrentLocationOnce(
...
+      manager_, &altitude, &latitude, &longitude, &climb, &direction, &speed,
+      &level, &horizontal, &vertical, &timestamp);
+  if (!ret) {
+    return ret;
+  }
+
+  location->longitude = longitude;
+  location->latitude = latitude;
+  location->timestamp = timestamp;
+  location->accuracy = horizontal;
```
Comment (HakkyuKim) [inline packages/geolocator/tizen/src/locaton_manager.cc:87]: I'm not familiar with the meanings of each variables, but it this correct?

Comment (bbrto21) [reply packages/geolocator/tizen/src/locaton_manager.cc:87]: Yes, both means to horizontal accuracy in meter.

File: packages/geolocator/tizen/src/geolocator_tizen_plugin.cc [hunk 2]
```diff
@@ -105,6 +107,18 @@ class GeolocatorTizenPlugin : public flutter::Plugin {
...
   }
 
+  void onGetLastKnownPosition(
+      std::unique_ptr<flutter::MethodResult<flutter::EncodableValue>> result) {
+    Location location;
+    TizenResult tizen_result =
+        location_manager_->GetLastKnownLocation(&location);
+    if (!tizen_result) {
+      result->Error("Failed to get last known position",
+                    tizen_result.message());
```
Comment (bwikbs) [inline packages/geolocator/tizen/src/geolocator_tizen_plugin.cc:117]: Shouldn't we return here?

**Comment by bbrto21** [comment]
@HakkyuKim I've update project_def.prop based on your review

File: packages/geolocator/tizen/src/geolocator_tizen_plugin.cc (hunk 2 above)
Comment (bbrto21) [reply packages/geolocator/tizen/src/geolocator_tizen_plugin.cc:117]: Thanks! I fixed it

**Review by bwikbs — APPROVED** [review]
Thanks!!

**Pull request merged**